examples/md-flexible/src/Simulation.cpp

```diff
@@ -329,3 +342,56 @@ void Simulation::updateThermostat() {
     _timers.thermostat.stop();
   }
 }
+
+long Simulation::accumulateTime(const long &time) {
+  long reducedTime;
+  autopas::AutoPas_MPI_Reduce(&time, &reducedTime, 1, AUTOPAS_MPI_LONG, AUTOPAS_MPI_SUM, 0, AUTOPAS_MPI_COMM_WORLD);
+
+  return reducedTime;
+}
+
+void Simulation::logTimers() {
+  long positionUpdate = accumulateTime(_timers.positionUpdate.getTotalTime());
+  long forceUpdateTotal = accumulateTime(_timers.forceUpdateTotal.getTotalTime());
+  long forceUpdatePairwise = accumulateTime(_timers.forceUpdatePairwise.getTotalTime());
+  long forceUpdateGlobal = accumulateTime(_timers.forceUpdateGlobal.getTotalTime());
+  long forceUpdateTuning = accumulateTime(_timers.forceUpdateTuning.getTotalTime());
+  long forceUpdateNonTuning = accumulateTime(_timers.forceUpdateNonTuning.getTotalTime());
+  long velocityUpdate = accumulateTime(_timers.velocityUpdate.getTotalTime());
+  long simulate = accumulateTime(_timers.simulate.getTotalTime());
+  long vtk = accumulateTime(_timers.vtk.getTotalTime());
+  long initialization = accumulateTime(_timers.initialization.getTotalTime());
+  long total = accumulateTime(_timers.total.getTotalTime());
+  long thermostat = accumulateTime(_timers.thermostat.getTotalTime());
+  long haloParticleExchange = accumulateTime(_timers.haloParticleExchange.getTotalTime());
+  long migratingParticleExchange = accumulateTime(_timers.migratingParticleExchange.getTotalTime());
+
+  if (_domainDecomposition.getDomainIndex() == 0) {
+    auto maximumNumberOfDigits = std::to_string(total).length();
+    std::cout << std::endl;
+    std::cout << timerToString("Total                      ", total, maximumNumberOfDigits, total);
+    std::cout << timerToString("Simulate                   ", simulate, maximumNumberOfDigits, total);
+    std::cout << timerToString("  Initialization           ", initialization, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  PositionUpdate           ", positionUpdate, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  VelocityUpdate           ", velocityUpdate, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("  ForceUpdateTotal         ", forceUpdateTotal, maximumNumberOfDigits, simulate);
+    std::cout << timerToString("    ForceUpdatePairwise    ", forceUpdatePairwise, maximumNumberOfDigits,
+                               forceUpdateTotal);
+    std::cout << timerToString("    ForceUdpateGlobal      ", forceUpdateGlobal, maximumNumberOfDigits,
```

Comment:
```suggestion
    std::cout << timerToString("    ForceUdpateGlobalF      ", forceUpdateGlobalF, maximumNumberOfDigits,
```
maybe better?

Reply:
In this case I would prefer ForceUpdateGlobalForce

Reply:
maybe plural just in case that there might come in more than one :)